app/controllers/application_controller.rb

```diff
@@ -85,7 +85,7 @@ def confirm_two_factor_authenticated
 
   def confirm_security_questions_setup
     return unless UserDecorator.new(current_user).needs_security_questions?(session)
-
+    flash.keep(:notice)
```

Comment:
Not sure what this is for, but it's not going to matter anyways since we're getting rid of security questions.